src/telemetry/metrics/transport.rs

```diff
@@ -23,6 +24,36 @@ pub(super) struct OpenMetrics {
 
 pub(super) type CloseScopes = Scopes<TransportCloseLabels, CloseMetrics>;
 
+/// Labels describing a TCP connection
+#[derive(Copy, Clone, Debug, Eq, PartialEq, Hash)]
+pub struct TransportLabels {
```

Comment:
TIOLI: Now that this lives in the `Transport` module, perhaps the name ought to be `transport::Labels`? Consumers in other modules can still `use transport::{Labels as TransportLabels}`.

Reply:
this is all going to change a lot more

Reply:
this is all going to change a lot more